<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ElogWindow</class>
 <widget class="QWidget" name="ElogWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Electronic logbook</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../../resources/nicos-gui.qrc">
    <normaloff>:/paste</normaloff>:/paste</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="margin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionBack">
   <property name="icon">
    <iconset resource="../../../../resources/nicos-gui.qrc">
     <normaloff>:/previous</normaloff>:/previous</iconset>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="icon">
    <iconset resource="../../../../resources/nicos-gui.qrc">
     <normaloff>:/next</normaloff>:/next</iconset>
   </property>
   <property name="text">
    <string>Forward</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="../../../../resources/nicos-gui.qrc">
     <normaloff>:/refresh</normaloff>:/refresh</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionAddComment">
   <property name="text">
    <string>Add comment</string>
   </property>
   <property name="toolTip">
    <string>Add a free-form comment entry to the logbook</string>
   </property>
  </action>
  <action name="actionAddRemark">
   <property name="text">
    <string>Set Remark</string>
   </property>
   <property name="toolTip">
    <string>Set the remark (instrument configuration header) and add it to the logbook</string>
   </property>
  </action>
  <action name="actionAttachFile">
   <property name="text">
    <string>Attach file</string>
   </property>
   <property name="toolTip">
    <string>Attach a file to the logbook</string>
   </property>
  </action>
  <action name="actionNewSample">
   <property name="text">
    <string>New Sample</string>
   </property>
   <property name="toolTip">
    <string>Enter a new sample name</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="../../../../resources/nicos-gui.qrc">
     <normaloff>:/print</normaloff>:/print</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="toolTip">
    <string>Print</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../../resources/nicos-gui.qrc"/>
 </resources>
 <connections/>
</ui>
